tests/test_operator.py

```diff
@@ -1,43 +0,0 @@
-import numpy as np
-from sympy import Eq
-
-from devito.dimension import t, x
-from devito.interfaces import DenseData
-from devito.operator import SimpleOperator
-
-
-class Test_Operator(object):
-    def test_codegen(self):
-        input_grid = DenseData(name="input_grid", shape=(3, 2), dtype=np.float64)
-        input_grid.data[:] = np.arange(6, dtype=np.float64).reshape((3, 2))
-        output_grid = DenseData(name="output_grid", shape=input_grid.shape,
-                                dtype=input_grid.dtype)
-        eq = Eq(output_grid.indexed[t, x], input_grid.indexed[t, x] + 3)
-        op = SimpleOperator(input_grid, output_grid, [eq])
-        op.apply()
-        assert(output_grid.data[2][1] == 8)
-
-    def test_python(self):
-        input_grid = DenseData(name="input_grid", shape=(3, 2), dtype=np.float64)
-        input_grid.data[:] = np.arange(6, dtype=np.float64).reshape((3, 2))
-        output_grid = DenseData(name="output_grid", shape=input_grid.shape,
-                                dtype=input_grid.dtype)
-        eq = Eq(output_grid.indexed[t, x], input_grid.indexed[t, x] + 3)
-        op = SimpleOperator(input_grid, output_grid, [eq])
-        op.apply(debug=True)
-        assert(output_grid.data[2][1] == 8)
-
-    def test_compare(self):
```

Comment:
I think this test is quite unique in that it is the only one that directly tests the correctness of the generated code against a "reference implementation". This should definitely be rewritten for the new operator.